pkg/runners/builtin.go

```diff
@@ -29,10 +29,10 @@ func (e *NotBuiltinCmd) Error() string {
 // Builtin runs the provided builtin command, or returns a
 // NotBuiltinCmd error if it is not one.
 func Builtin(command string, args []string) error {
```

Comment:
Later on this could be removed, I kept it because I need to leave soon for something but would look into this after I finish the lexer 😄